packages/nc-gui/components/smartsheet/column/SelectOptions.vue

```diff
@@ -184,6 +179,16 @@ watch(inputs, () => {
     inputs.value.$el.focus()
   }
 })
+
+// Removes the Select Option from cdf if the option is removed
+watch(vModel.value, (next) => {
+  const cdfs = (next.cdf ?? '').split(',')
+  const values = (next.colOptions.options ?? []).map((col) => {
+    return col.title.replaceAll("'", '')
```

Comment:
Do we have to replace all the quotes or just the leading and trailing ?

Reply:
Updated👍